contents/docs/cdp/batch-exports/postgres.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Postgres destination for batch exports
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+Batch exports can be used to export data to a Postgres table.
+
+## Setting up Postgres access
+
+1. Make sure PostHog can access your Postgres database.
+
+> **Note:** Wherever your Postgres database is hosted, make sure the host is set to accept all incoming connections so that PostHog can connect to the database and insert events. If this is not possible in your case, consider exporting data to a different destination (like [S3](./s3.md)) and then setting up your own system for getting data into your Postgres database.
+
+2. Create a Postgres user with table creation privileges.
```

Comment:
```suggestion
Make sure PostHog can access your Postgres database by creating a Postgres user with table creation privileges.

> **Note:** Wherever your Postgres database is hosted, make sure the host is set to accept all incoming connections so that PostHog can connect to the database and insert events. If this is not possible, consider exporting data to a different destination (like [S3](/docs/cdp/batch-exports/s3)) and then setting up your own system for getting data into your Postgres database.
```